Zlib.autopkg

```diff
@@ -22,18 +22,22 @@ nuget
       summary: @"Zlib Compression Library for use in TechSmith products";
       description: @"Zlib is a compression library written in C.
          New features:
-         Breaking changes:
          Bug fixes:
-         Misc:
-            - Updated toolset to v141 for Visual Studio 2017
+         Breaking changes:
+         Changes:
+            - 1.2.11.3 Include contrib/minizip header files
+            - 1.2.11.2 Updated toolset to v141 for Visual Studio 2017
+            - 1.2.11.1 Removed ASMV & ASMINF from Release 
+            - 1.2.11   Initial version
          ";
       copyright: @"Copyright (c) 2018.";
       tags: { zlib, CoApp, compression, vs2017, native };
    };
 
    files
    {
-      nestedInclude: { #destination=${d_include}Zlib; *.h };
+      nested2Include: { #destination=${d_include}Zlib; "*.h" };
+      nested1Include: { #destination=${d_include}contrib\minizip; "*.h" };
```

Comment:
This is not correct. It means that all headers should go under `contrib\minizip`
![image](https://user-images.githubusercontent.com/5627161/121660545-a7492200-caa3-11eb-9af5-8b5f75781a12.png)
(BTW you can open up the nuget package with 7zip, no need to install the package)

Reply:
I think it should be:
`{ #destination=${d_include}Zlib\contrib\minizip; "contrib\minizip\*.h" };`
(untested)
We want the contrib to go under Zlib, so that the consumer does:
`#include <ZLib/contrib/minizip/ioapi.h>`
instead of
`#include <contrib/minizip/ioapi.h>` <<<< not Zlib specific, it does not look good.

Reply:
🤦 I thought I saw the correct ones there before - I'm sorry waste your time with this one. Re 7zip, yeah, I've got *.nupkg associated with 7zip. Fixed nupkg is [here](https://drive.google.com/drive/folders/1kvNZMQmgX5vOm0FutWcAeb-4ew8qwcN5?usp=sharing).

Reply:
Thanks for the update and np on me trying out that version. Will test it on Monday!
Was it built on your machine?

One thing that I still have to look into is that pesky warning:
`warning LNK4098: defaultlib 'libcmt.lib' conflicts with use of other libs; use /NODEFAULTLIB:library`
Not exactly sure what project is "at fault" and how I can solve this but I will look on Monday

Reply:
Yes, built on my machine. Time permitting, I'll try your branch and this nupkg on my branch today, and see if I can learn anything about that warning.